pkg/controller/controller_broker.go

```diff
@@ -113,8 +113,8 @@ const (
 // shouldReconcileServiceBroker determines whether a broker should be reconciled; it
 // returns true unless the broker has a ready condition with status true and
 // the controller's broker relist interval has not elapsed since the broker's
-// ready condition became true.
-func shouldReconcileServiceBroker(broker *v1alpha1.ServiceBroker, now time.Time, relistInterval time.Duration) bool {
+// ready condition became true, or if the broker's RelistBehavior is set to Manual
+func shouldReconcileServiceBroker(broker *v1alpha1.ServiceBroker, now time.Time, relistInterval metav1.Duration) bool {
```

Comment:
Technically you do not need to pass in the `relistInterval` parameter here, since you're now reading it from the broker.

Reply:
Oh, I didn't realize you were still reading it as a parameter.  You already have the broker here - you can read the value in this method and delete the `relistInterval` parameter.  There needs to ve a validation that ensures that the duration field is set when the mode is 'duration'.